multiqc/modules/bamqc/assets/js/multiqc_bamqc.js

```diff
@@ -19,17 +19,19 @@ $(function () {
         var fail_percent = (v['fail']/total)*100;
         var pass_number_of_digits = v['pass'].toString().length;
         var fail_number_of_digits = v['fail'].toString().length;
+        var pass = pass_percent; // variable used to specify the width of a progress-bar-pass 
+        var fail = fail_percent; // vairable used to specify the width of a progress-bar-fail
         // Each digit needs around 8px of space
         if (pass_percent != 0 && pass_percent < pass_number_of_digits * 8) {
-            pass_percent = pass_number_of_digits * 8;
-            fail_percent = 100 - pass_number_of_digits * 8;
+            pass = pass_number_of_digits * 8;
+            fail = 100 - pass_number_of_digits * 8;
         } else if (fail_percent != 0 && fail_percent < fail_number_of_digits * 8) {
-            fail_percent = fail_number_of_digits * 8;
-            pass_percent = 100 - fail_number_of_digits * 8;
+            fail = fail_number_of_digits * 8;
+            pass = 100 - fail_number_of_digits * 8;
         }
         var p_bar = '<div class="progress bamqc_passfail_progress"> \
-            <div class="progress-bar progress-bar-info" style="width: '+pass_percent+'%" title="'+v['pass']+'&nbsp;/&nbsp;'+total+' samples passed">'+v['pass']+'</div> \
-            <div class="progress-bar progress-bar-warning" style="width: '+fail_percent+'%" title="'+v['fail']+'&nbsp;/&nbsp;'+total+' samples are more than two standard deviations away from the mean">'+v['fail']+'</div> \
+            <div class="progress-bar progress-bar-info" style="width: '+pass+'%" title="'+v['pass']+'&nbsp;/&nbsp;'+total+' samples passed">'+v['pass']+'</div> \
```

Comment:
On line 24 you say "each digit needs around 8px of space" but here you're setting pass to a %, not to a number of pixels. Could you please clarify which is correct?

Reply:
I am sorry about the confusion it caused. So the entire passfail_progress_bar is set to have a total width of 100px, and the width for either samples that are passed or samples that are failed will be their corresponding percentage times the total width (for example, if pass is 16%, it will have a width of 16% * 100px = 16px). By this, "each digit needs around 8px of space" is equivalent to "each digit needs around 8% of space".

Reply:
Ok, that's relevant context to include in your comment on line 24.